tests/acceptance/api_test.go

```diff
@@ -2173,12 +2224,21 @@ func apiRequest(t *testing.T, input *apiRequestInput) *apiResponse {
 	return apiResp
 }
 
-func parseResponseJSON(t *testing.T, resp *apiResponse) map[string]interface{} {
+func parseResponseJSON(t require.TestingT, resp *apiResponse) map[string]interface{} {
```

Comment:
changed the interface here, so we can use these methods in the context of  `assert`, `require`, and `retry` calls